build/configs/rtl8730e/loadable_ext/defconfig

```diff
@@ -501,7 +501,12 @@ CONFIG_SPI_EXCHANGE=y
 
 #
 # LCD Driver Support
-#
+CONFIG_LCD=y
+CONFIG_LCD_ST7785=y
```

Comment:
ST7785 and ST7785M are the same?

Reply:
I have used ST7785 although the lcd is ST7785M. should I use ST7785M in configs?